content/en/blog/2022-10-14-unikraft-community-meetup.md

```diff
@@ -0,0 +1,85 @@
+
+### A Hike, a Grill and a Presentation Walk into a Bar
+
+What do all these have in common, you ask? Well, one might be tempted to say nothing, but at a closer look this is what the main
+day of the trip looked like.
+
+After a late arrival in Sinaia our participants had a somewhat early and uneventful end of the day.
+Things took a more interesting turn when our participants went for a hike early in the morning (for those of you who, like me, can't believe it,
+we will have pictures at the end to prove it, so tag along). Some went for a more traditional hike at
+[Vf. Piscul Cainelui](https://muntii-nostri.ro/ro/routeinfo/sinaia-vf-piscul-cainelui), while others went for a more
+conservative visit of the local places, like the [Peles and Pelisor](https://peles.ro/) castle.
+
+Sometime at midday both groups returned to the accommodations, where the main event was about to start, a community contributors presentation, which was composed of five topics.
+
+The first one to break the ice was our community manager, [Răzvan Deaconescu](https://www.linkedin.com/in/razvandeaconescu/),
+who talked us through the general state of the community.
+The highlight of the presentation was the ever incresing number of github stars, recently breaking the thousand mark.
+For more details, we have a blogpost about it [here](https://unikraft.org/blog/2022-10-14-unikraft-reaches-1k-github-stars/).
+
+Coming immediately after, we had [Alexander Jung](https://www.linkedin.com/in/nderjung/), CPO at Unikraft, to give us a small talk about the current state of kraftkit, the future of the kraft companion for building unikernels, which we
+hope to deliver in the next release. Tune in for more details because we will for sure have a blogpost about it.
+
+Afterward, we had [Simon Kuenzer](https://www.linkedin.com/in/simon-kuenzer-690a3145/), CTO and Co-Founder, give us a presentation about musl support.
+Our aim through musl is to have a better binary compatibility, since
+musl is the closest to libc.
+We aim to replace newlibc and pthread-embedded for more complex application which use threads, while newlibc will remain as an
+alternative to simple, single threaded applications.
+
+Continuing the trend, we had [Hugo Lefeuvre](https://www.linkedin.com/in/hugo-lefeuvre-at-debian/), core contributor to Unikraft, to give us a more detailed presentation about the way Unikraft handles security and how we want to ever increase it in our unikernels.
+Here, a great deal of currently ongoing projects have been discussed. We don't want to spoil it for you, but we can give a few names like `ASLR`,  `Shadow Stacks` and `W^X`. As you might have already guessed, we will try to dedicate a future blogpost to each.
+
+Last but not least, we had [Michalis Pappas](https://www.linkedin.com/in/michalispappas/), core contributor, whose discussion approached
+the subject of platform rearchitecting and how it can improve Unikraft.
```

Comment:
```suggestion
Last but not least, we had [Michalis Pappas](https://www.linkedin.com/in/michalispappas/), core contributor, whose discussion approached the subject of platform rearchitecting and how it can improve Unikraft.
```